scripts/dappnode_install.sh

```diff
@@ -290,6 +323,30 @@ installExtraDpkg() {
     fi
 }
 
+# The main user needs to be added to the docker group to be able to run docker commands without sudo
+# Explained in: https://docs.docker.com/engine/install/linux-postinstall/
```

Comment:
we should stop relying on upstream documentation that may change at any time and start using something dynamic like https://github.com/docker/docker-install/tree/master